dietpi/dietpi-software

```diff
@@ -1076,6 +1076,16 @@ _EOF_
 					  aSOFTWARE_TYPE[$index_current]=0
 			 aSOFTWARE_ONLINEDOC_URL[$index_current]='f=8&t=5&p=7575#p7575'
 
+		#------------------
+		index_current=155
```

Comment:
Should be 155 everywhere. It was 152 when I was testing. Fixing for next iteration.